airflow/dags/copy_prod_archiver_configs_to_test/copy_prod_archiver_configs_to_test.py

```diff
@@ -0,0 +1,55 @@
+from calitp_data_infra.storage import GTFSDownloadConfigExtract, get_latest
+from google.cloud import storage
+
+prod_bucket = "calitp-gtfs-download-config"
```

Comment:
I would consider defining these as environment variables in the docker-compose YAML so that they're configurable